# translation of update-manager to Spanish
# This file is distributed under the same license as the update-manager package.
#  Copyright (c) 2004 Canonical
#                2004 Michiel Sikkes
# Jorge Bernal <koke@amedias.org>, 2005.
# Jorge Bernal <koke@sindominio.net>, 2005.
# Paco Molinero <paco@byasl.com>, 2011.
#
#: ../DistUpgrade/DistUpgradeController.py:1075
msgid ""
msgstr ""
"Project-Id-Version: es\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2020-11-17 15:33-0800\n"
"PO-Revision-Date: 2019-06-18 17:52+0000\n"
"Last-Translator: Paco Molinero <paco@byasl.com>\n"
"Language-Team: Spanish <traductores@gnome.org>\n"
"Language: es\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2019-11-15 04:48+0000\n"
"X-Generator: Launchpad (build c597c3229eb023b1e626162d5947141bf7befb13)\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:215 ../DistUpgrade/distro.py:446
#, python-format
msgid "Server for %s"
msgstr "Servidor para %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:233 ../DistUpgrade/distro.py:239
#: ../DistUpgrade/distro.py:255
msgid "Main server"
msgstr "Servidor principal"

#: ../DistUpgrade/distro.py:259
msgid "Custom servers"
msgstr "Servidores personalizados"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:151
msgid "Could not calculate sources.list entry"
msgstr "No se puede calcular la entrada en sources.list"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:260
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"No se pudo localizar ningún paquete, quizá no es un disco de Ubuntu o no es "
"la arquitectura correcta."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:310
msgid "Failed to add the CD"
msgstr "Error al añadir el CD"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:311
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Ha habido un error al añadir el CD; se ha interrumpido la actualización. "
"Informe de esto como un fallo si este es un CD válido de Ubuntu.\n"
"\n"
"El mensaje de error fue:\n"
"«%s»"

#: ../DistUpgrade/DistUpgradeCache.py:182
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Desinstalar paquete en mal estado"
msgstr[1] "Desinstalar paquetes en mal estado"

#: ../DistUpgrade/DistUpgradeCache.py:185
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"El paquete «%s» está en un estado inconsistente y debe reinstalarse, pero no "
"se encuentra en ningún repositorio. ¿Quiere desinstalar este paquete ahora "
"para continuar?"
msgstr[1] ""
"Los paquetes «%s» están en un estado inconsistente y deben reinstalarse, "
"pero no se encuentran en ningún repositorio. ¿Quiere desinstalar estos "
"paquetes ahora para continuar?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:301
msgid "The server may be overloaded"
msgstr "El servidor puede estar sobrecargado"

#: ../DistUpgrade/DistUpgradeCache.py:414
msgid "Broken packages"
msgstr "Paquetes rotos"

#: ../DistUpgrade/DistUpgradeCache.py:415
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Su sistema contiene paquetes rotos que no se pueden reparar con este "
"software. Repárelos primero mediante Synaptic o apt-get antes de continuar."

#: ../DistUpgrade/DistUpgradeCache.py:660
#, fuzzy, python-format
msgid "Broken packages after upgrade: %s"
msgstr "Paquetes rotos"

#. the most likely problem is the 3rd party pkgs so don't address
#. foreignPkgs and devRelease being True
#: ../DistUpgrade/DistUpgradeCache.py:669
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" "
msgstr ""
"Se produjo un problema irresoluble al calcular la actualización.\n"
"\n"
" "

#: ../DistUpgrade/DistUpgradeCache.py:672
msgid ""
"This was likely caused by:\n"
" * Unofficial software packages not provided by Ubuntu\n"
"Please use the tool 'ppa-purge' from the ppa-purge \n"
"package to remove software from a Launchpad PPA and \n"
"try the upgrade again.\n"
"\n"
msgstr ""
"Esto podría deberse a:\n"
" * Paquetes de software no oficiales, no proporcionados por Ubuntu\n"
"Utilice la herramienta «ppa-purge» desde el paquete \n"
"ppa-purge para quitar software de una PPA de Launchpad PPA e \n"
"intente actualizar de nuevo.\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:680
msgid ""
"This was caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
"This is most likely a transient problem, \n"
"please try again later.\n"
msgstr ""
"Esto fue causado por:\n"
" * Actualizar a una versión de prelanzamiento de Ubuntu\n"
"Probablemente sea un problema transitorio, \n"
"intente de nuevo más tarde.\n"

#: ../DistUpgrade/DistUpgradeCache.py:687
msgid "This is most likely a transient problem, please try again later."
msgstr ""
"Probablemente sea un problema transitorio, inténtelo de nuevo más tarde."

#: ../DistUpgrade/DistUpgradeCache.py:690
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal. "
msgstr ""
"Si nada de esto es aplicable, entonces informe de este error usando el "
"comando «ubuntu-bug ubuntu-release-upgrader-core» en una terminal. "

#: ../DistUpgrade/DistUpgradeCache.py:692
msgid ""
"If you want to investigate this yourself the log files in '/var/log/dist-"
"upgrade' will contain details about the upgrade. Specifically, look at 'main."
"log' and 'apt.log'."
msgstr ""
"Si quiere investigar el problema por su cuenta, los registros en «/var/log/"
"dist-upgrade» contienen detalles de la actualización; en particular, "
"consulte «main.log» y «apt.log»."

#: ../DistUpgrade/DistUpgradeCache.py:698
msgid "Could not calculate the upgrade"
msgstr "No se ha podido calcular la actualización"

#: ../DistUpgrade/DistUpgradeCache.py:757
msgid "Error authenticating some packages"
msgstr "Error al autenticar algunos paquetes"

#: ../DistUpgrade/DistUpgradeCache.py:758
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"No ha sido posible autenticar algunos paquetes. Esto puede ser debido a un "
"problema transitorio en la red. Pruebe de nuevo más tarde. Vea abajo una "
"lista de los paquetes no autenticados."

#: ../DistUpgrade/DistUpgradeCache.py:779
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"El paquete «%s» está marcado para desinstalarse pero está en la lista negra "
"de desinstalación."

#: ../DistUpgrade/DistUpgradeCache.py:785
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "El paquete esencial «%s» ha sido marcado para su desinstalación."

#: ../DistUpgrade/DistUpgradeCache.py:794
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Intentando instalar la versión prohibida «%s»"

#: ../DistUpgrade/DistUpgradeCache.py:916
#, python-format
msgid "Can't install '%s'"
msgstr "No se ha podido instalar «%s»"

#: ../DistUpgrade/DistUpgradeCache.py:917
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"Fue imposible instalar un paquete requerido. Informe de esto como un error "
"usando «ubuntu-bug ubuntu-release-upgrader-core» en una terminal."

#: ../DistUpgrade/DistUpgradeCache.py:928
msgid "Can't guess meta-package"
msgstr "No se ha podido determinar el metapaquete"

#: ../DistUpgrade/DistUpgradeCache.py:929
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Su sistema no incluye un paquete %s o %s y no ha sido posible detectar qué "
"versión de Ubuntu está ejecutando.\n"
" Instale uno de los paquetes anteriores usando synaptic o apt-get antes de "
"continuar."

#: ../DistUpgrade/DistUpgradeController.py:113
msgid "Reading cache"
msgstr "Leyendo la caché"

#: ../DistUpgrade/DistUpgradeController.py:225
msgid "Unable to get exclusive lock"
msgstr "No se ha podido obtener un bloqueo exclusivo"

#: ../DistUpgrade/DistUpgradeController.py:226
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Esto normalmente significa que ya se está ejecutando otra aplicación de "
"gestión de paquetes (como apt-get o aptitude). Cierre esa aplicación primero."

#: ../DistUpgrade/DistUpgradeController.py:284
msgid "Upgrading over remote connection not supported"
msgstr "Actualizando sobre conexión remota no compatible"

#: ../DistUpgrade/DistUpgradeController.py:285
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Está ejecutando la actualización sobre una conexión ssh remota con una "
"interfaz de usuario que no lo permite. Intente actualizar en modo texto con "
"«do-release-upgrade».\n"
"\n"
"La actualización se detendrá ahora. Pruebe sin ssh."

#: ../DistUpgrade/DistUpgradeController.py:299
msgid "Continue running under SSH?"
msgstr "¿Continuar la ejecución bajo SSH?"

#: ../DistUpgrade/DistUpgradeController.py:300
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Esta sesión parece estar ejecutándose bajo ssh. No es recomendable hacer una "
"actualización sobre ssh actualmente, porque en caso de fallo es muy difícil "
"recuperarla.\n"
"\n"
"Si continua, se iniciará un demonio ssh adicional en el puerto «%s».\n"
"¿Quiere continuar?"

#: ../DistUpgrade/DistUpgradeController.py:314
msgid "Starting additional sshd"
msgstr "Iniciando sshd adicional"

#: ../DistUpgrade/DistUpgradeController.py:315
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Para facilitar la recuperación en caso de fallo, se iniciará un sshd "
"adicional en el puerto «%s». Si algo va mal con el ssh en ejecución, aún "
"podrá conectarse al adicional.\n"

#: ../DistUpgrade/DistUpgradeController.py:323
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Si ejecuta un cortafuegos, puede necesitar abrir este puerto temporalmente. "
"Como esto es potencialmente peligroso, no se hace automáticamente. Puede "
"abrir el puerto con:\n"
"«%s»"

#: ../DistUpgrade/DistUpgradeController.py:401
#: ../DistUpgrade/DistUpgradeController.py:425
msgid "Can not upgrade"
msgstr "No se puede actualizar"

#: ../DistUpgrade/DistUpgradeController.py:402
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Esta herramienta no permite actualizaciones de «%s» a «%s»."

#: ../DistUpgrade/DistUpgradeController.py:426
msgid ""
"Your python3 install is corrupted. Please fix the '/usr/bin/python3' symlink."
msgstr ""
"Su instalación de python3 está dañada. Repare el enlace simbólico «/usr/bin/"
"python3»."

#: ../DistUpgrade/DistUpgradeController.py:455
#, python-format
msgid "Can not write to '%s'"
msgstr "No se puede escribir en «%s»"

#: ../DistUpgrade/DistUpgradeController.py:456
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"No es posible escribir el directorio de sistema «%s» en su sistema. La "
"actualización no puede continuar.\n"
"Asegúrese de que el directorio de sistema permite escribir."

#: ../DistUpgrade/DistUpgradeController.py:467
msgid "Include latest updates from the Internet?"
msgstr "¿Incluir las últimas actualizaciones desde Internet?"

#: ../DistUpgrade/DistUpgradeController.py:468
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"El sistema de actualización a una nueva versión puede usar Internet para "
"descargar automáticamente las actualizaciones más recientes e instalarlas "
"durante el proceso. Si dispone de una conexión a la red, esto es muy "
"recomendable.\n"
"\n"
"La actualización llevará más tiempo, pero una vez terminada, el sistema "
"estará completamente actualizado. Puede elegir no hacerlo, pero deberá "
"instalar las nuevas actualizaciones inmediatamente después de pasar a la "
"nueva versión.\n"
"Si responde «no» ahora, la red no se usará para nada."

#: ../DistUpgrade/DistUpgradeController.py:646
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr ""
"No está destinado a usuarios durante la etapa de desarrollo de la versión %s"

#: ../DistUpgrade/DistUpgradeController.py:747
#, python-format
msgid "disabled on upgrade to %s"
msgstr "(desactivado al actualizar a %s)"

#: ../DistUpgrade/DistUpgradeController.py:806
msgid "No valid sources.list entry found"
msgstr "No se ha encontrado una entrada sources.list válida"

#: ../DistUpgrade/DistUpgradeController.py:807
#, python-format
msgid ""
"While scanning your repository information no entry about %s could be "
"found.\n"
"\n"
"An upgrade might not succeed.\n"
"\n"
"Do you want to continue anyway?"
msgstr ""
"Al escanear su información de repositorio no se pudo encontrar ninguna "
"entrada relacionada con %s.\n"
"\n"
"La actualización podría fracasar.\n"
"\n"
"¿Desea continuar de todas formas?"

#: ../DistUpgrade/DistUpgradeController.py:817
msgid "No valid mirror found"
msgstr "No se ha encontrado un servidor espejo válido"

#: ../DistUpgrade/DistUpgradeController.py:818
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Mientras se analizaba la información de los repositorios, no se encontró una "
"entrada de servidor espejo para la actualización. Esto puede ocurrir si "
"tiene su propio servidor interno o si la información no está actualizada.\n"
"\n"
"¿Quiere reescribir el archivo «sources.list» de todos modos? Si elige «Sí» "
"se actualizarán todas las entradas «%s» a «%s».\n"
"Si elige «No», se cancelará la actualización."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:838
msgid "Generate default sources?"
msgstr "¿Generar orígenes predeterminados?"

#: ../DistUpgrade/DistUpgradeController.py:839
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Después de explorar su «sources.list» no se encontraron entradas válidas "
"para «%s».\n"
"\n"
"¿Se deben añadir entradas predeterminadas para «%s»? Si selecciona «No», la "
"actualización se cancelará."

#: ../DistUpgrade/DistUpgradeController.py:874
#: ../DistUpgrade/DistUpgradeController.py:880
msgid "Repository information invalid"
msgstr "Información de repositorio no válida"

#: ../DistUpgrade/DistUpgradeController.py:875
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"La actualización de la información del repositorio dio como resultado un "
"archivo no válido por lo que se está iniciando un proceso de notificación de "
"errores."

#: ../DistUpgrade/DistUpgradeController.py:881
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"Como resultado de actualizar la información del repositorio se obtuvo un "
"archivo no válido. Para informar de un error instale apport y después "
"ejecute «apport-bug ubuntu-release-upgrader»."

#: ../DistUpgrade/DistUpgradeController.py:890
msgid "Third party sources disabled"
msgstr "Desactivados los orígenes de terceros"

#: ../DistUpgrade/DistUpgradeController.py:891
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Se han desactivado algunas entradas de otros proveedores en su «sources."
"list». Puede volver a activarlas tras la actualización con la herramienta "
"«Orígenes del software», o con su gestor de paquetes."

#: ../DistUpgrade/DistUpgradeController.py:933
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Paquete en un estado inconsistente"
msgstr[1] "Paquetes en un estado inconsistente"

#: ../DistUpgrade/DistUpgradeController.py:936
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"El paquete «%s» está en un estado inconsistente y debe reinstalarse, pero no "
"se encuentra en ningún repositorio. Reinstale el paquete manualmente o "
"desinstálelo del sistema."
msgstr[1] ""
"Los paquetes «%s» están en un estado inconsistente y deben reinstalarse, "
"pero no se encuentran en ningún repositorio. Reinstale los paquetes "
"manualmente o desinstálelos del sistema."

#: ../DistUpgrade/DistUpgradeController.py:999
msgid "Error during update"
msgstr "Error durante la actualización"

#: ../DistUpgrade/DistUpgradeController.py:1000
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Ocurrió un problema durante la actualización. Normalmente es debido a algún "
"tipo de problema en la red, por lo que le recomendamos que compruebe su "
"conexión de red y vuelva a intentarlo."

#: ../DistUpgrade/DistUpgradeController.py:1043
#: ../DistUpgrade/DistUpgradeController.py:1048
msgid "EFI System Partition (ESP) not usable"
msgstr "Partición de sistema EFI (ESP) no utilizable"

#: ../DistUpgrade/DistUpgradeController.py:1044
msgid ""
"Your EFI System Partition (ESP) is not mounted at /boot/efi. Please ensure "
"that it is properly configured and try again."
msgstr ""
"La partición de sistema EFI (ESP) no está montada en /boot/efi. Cerciórese "
"de que esté configurada adecuadamente y vuelva a intentarlo."

#: ../DistUpgrade/DistUpgradeController.py:1049
msgid ""
"The EFI System Partition (ESP) mounted at /boot/efi is not writable. Please "
"mount this partition read-write and try again."
msgstr ""
"No se puede escribir en la partición de sistema EFI (ESP) montada en /boot/"
"efi. Monte la partición en modo lectura/escritura y vuelva a intentarlo."

#: ../DistUpgrade/DistUpgradeController.py:1056
msgid "Not enough free disk space"
msgstr "No hay espacio suficiente en el disco"

#. TRANSLATORS: you can change the order of the sentence,
#. make sure to keep all {str_*} string untranslated.
#: ../DistUpgrade/DistUpgradeController.py:1059
#, python-brace-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of {str_total} free space "
"on disk '{str_dir}'. Please free at least an additional {str_needed} of disk "
"space on '{str_dir}'. {str_remedy}"
msgstr ""
"Se ha abortado la actualización. La actualización necesita un total de "
"{str_total} de espacio libre en el disco «{str_dir}». Libere al menos "
"{str_needed} de espacio libre adicional en «{str_dir}». {str_remedy}"

#. specific ways to resolve lack of free space
#: ../DistUpgrade/DistUpgradeController.py:1064
msgid ""
"Remove temporary packages of former installations using 'sudo apt clean'."
msgstr ""
"Elimine los paquetes temporales de instalaciones anteriores mediante «sudo "
"apt clean»."

#: ../DistUpgrade/DistUpgradeController.py:1066
msgid ""
"You can remove old kernels using 'sudo apt autoremove' and you could also "
"set COMPRESS=xz in /etc/initramfs-tools/initramfs.conf to reduce the size of "
"your initramfs."
msgstr ""
"Puede eliminar núcleos antiguos usando «sudo apt autoremove» y también puede "
"establecer COMPRESS=xz en /etc/initramfs-tools/initramfs.conf para reducir "
"el tamaño de su initramfs."

#: ../DistUpgrade/DistUpgradeController.py:1071
msgid ""
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""
"Vacíe la papelera y elimine los paquetes temporales de instalaciones "
"anteriores haciendo «sudo apt clean»."

#: ../DistUpgrade/DistUpgradeController.py:1074
msgid "Reboot to clean up files in /tmp."
msgstr "Reinicie para vaciar la carpeta /tmp."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1113
#: ../DistUpgrade/DistUpgradeController.py:1997
#: ../DistUpgrade/DistUpgradeController.py:2047
msgid "Calculating the changes"
msgstr "Calculando los cambios"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1157
msgid "Do you want to start the upgrade?"
msgstr "¿Quiere comenzar la actualización?"

#: ../DistUpgrade/DistUpgradeController.py:1176
#, python-format
msgid ""
"Livepatch security updates are not available for Ubuntu %s. If you upgrade, "
"Livepatch will turn off."
msgstr ""
"Las actualizaciones de seguridad de Livepatch no están disponibles para "
"Ubuntu %s. Si actualiza, Livepatch se apagará."

#: ../DistUpgrade/DistUpgradeController.py:1248
msgid "Upgrade canceled"
msgstr "Actualización cancelada"

#: ../DistUpgrade/DistUpgradeController.py:1249
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"La actualización se cancelará ahora y el sistema volverá a su estado "
"original. Puede reanudar la actualización posteriormente."

#: ../DistUpgrade/DistUpgradeController.py:1255
#: ../DistUpgrade/DistUpgradeController.py:1400
msgid "Could not download the upgrades"
msgstr "No se han podido descargar las actualizaciones"

#: ../DistUpgrade/DistUpgradeController.py:1256
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"La actualización se ha anulado. Compruebe su conexión a Internet o los "
"medios de instalación y vuelva a intentarlo. Todos los archivos descargados "
"hasta el momento se han mantenido."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1349
#: ../DistUpgrade/DistUpgradeController.py:1388
#: ../DistUpgrade/DistUpgradeController.py:1504
msgid "Error during commit"
msgstr "Error durante la confirmación"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1351
#: ../DistUpgrade/DistUpgradeController.py:1390
#: ../DistUpgrade/DistUpgradeController.py:1551
msgid "Restoring original system state"
msgstr "Restaurando el estado original del sistema"

#: ../DistUpgrade/DistUpgradeController.py:1352
#: ../DistUpgrade/DistUpgradeController.py:1367
#: ../DistUpgrade/DistUpgradeController.py:1391
msgid "Could not install the upgrades"
msgstr "No se han podido instalar las actualizaciones"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1357
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"La actualización se ha cancelado. Su sistema podría haber quedado en un "
"estado no utilizable. Ahora se llevará a cabo una recuperación (dpkg --"
"configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1362
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug and attach the files in /var/log/"
"dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Informe de este error usando un navegador para ir a http://bugs.launchpad."
"net/ubuntu/+source/ubuntu-release-upgrader/+filebug y adjunte los archivos "
"en /var/log/dist-upgrade/ al informe.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1401
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"La actualización se ha cancelado. Compruebe su conexión a Internet o el "
"soporte de instalación y vuelva a intentarlo. "

#: ../DistUpgrade/DistUpgradeController.py:1429
msgid "Searching for obsolete software"
msgstr "Buscando paquetes obsoletos"

#: ../DistUpgrade/DistUpgradeController.py:1492
msgid "Remove obsolete packages?"
msgstr "¿Desinstalar los paquetes obsoletos?"

#: ../DistUpgrade/DistUpgradeController.py:1493
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Conservar"

#: ../DistUpgrade/DistUpgradeController.py:1493
msgid "_Remove"
msgstr "_Eliminar"

#: ../DistUpgrade/DistUpgradeController.py:1505
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Ha ocurrido algún problema durante el limpiado. Lea el mensaje siguiente "
"para tener más información. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1589
msgid "Required depends is not installed"
msgstr "La dependencia requerida no está instalada"

#: ../DistUpgrade/DistUpgradeController.py:1590
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "La dependencia requerida «%s» no está instalada. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1857
#: ../DistUpgrade/DistUpgradeController.py:1947
msgid "Checking package manager"
msgstr "Comprobando el gestor de paquetes"

#: ../DistUpgrade/DistUpgradeController.py:1863
#: ../DistUpgrade/DistUpgradeController.py:1869
msgid "Preparing the upgrade failed"
msgstr "Falló la preparación de la actualización"

#: ../DistUpgrade/DistUpgradeController.py:1864
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""
"Falló la preparación del sistema para actualizarse, de manera que se ha "
"iniciado el proceso de informe de errores."

#: ../DistUpgrade/DistUpgradeController.py:1870
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"Falló al preparar el sistema para la actualización. Para informar de un "
"error instale apport y después ejecute «apport-bug ubuntu-release-upgrader»."

#: ../DistUpgrade/DistUpgradeController.py:1887
#: ../DistUpgrade/DistUpgradeController.py:1897
msgid "Getting upgrade prerequisites failed"
msgstr "Ha fallado la obtención de los requisitos previos de la actualización"

#: ../DistUpgrade/DistUpgradeController.py:1888
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"El sistema fue incapaz de obtener los requisitos previos para la "
"actualización. La actualización se interrumpirá ahora y se restaurará el "
"estado original del sistema.\n"
"\n"
"Adicionalmente, se está iniciando un proceso de notificación de errores."

#: ../DistUpgrade/DistUpgradeController.py:1898
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""
"El sistema no fue capaz de obtener los prerrequisitos para la actualización. "
"Se va a cancelar ahora la actualización y se restaurará el estado original "
"del sistema.\n"
"\n"
"Para informar de un error instale apport y después ejecute «apport-bug "
"ubuntu-release-upgrader»."

#: ../DistUpgrade/DistUpgradeController.py:1931
msgid "Updating repository information"
msgstr "Actualizando la información del repositorio"

#: ../DistUpgrade/DistUpgradeController.py:1938
msgid "Failed to add the cdrom"
msgstr "Fallo al añadir el CDROM"

#: ../DistUpgrade/DistUpgradeController.py:1939
msgid "Sorry, adding the cdrom was not successful."
msgstr "Lo sentimos, no se pudo añadir el CDROM"

#: ../DistUpgrade/DistUpgradeController.py:1974
msgid "Invalid package information"
msgstr "Información sobre los paquetes no válida"

#: ../DistUpgrade/DistUpgradeController.py:1975
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""
"Después de actualizar la información de paquete, el paquete esencial «%s» no "
"se puede localizar. Esto puede deberse a que no tiene las réplicas oficiales "
"listadas en sus de orígenes de software o por una carga excesiva en la "
"réplica que está usando. Vea /etc/apt/sources.list para conocer la "
"configuración de lista actual de orígenes de software.\n"
"En caso de tratarse de una sobrecarga de la réplica, debe esperar para "
"reintentar la actualización más tarde."

#: ../DistUpgrade/DistUpgradeController.py:2004
#: ../DistUpgrade/DistUpgradeController.py:2103
msgid "Fetching"
msgstr "Descargando"

#: ../DistUpgrade/DistUpgradeController.py:2011
#: ../DistUpgrade/DistUpgradeController.py:2032
#: ../DistUpgrade/DistUpgradeController.py:2053
#: ../DistUpgrade/DistUpgradeController.py:2107
msgid "Upgrading"
msgstr "Actualizando"

#: ../DistUpgrade/DistUpgradeController.py:2013
msgid "Upgrade infeasible"
msgstr "Actualización inviable"

#: ../DistUpgrade/DistUpgradeController.py:2014
msgid ""
"The upgrade could not be completed, there were errors during the upgrade "
"process."
msgstr ""
"La actualización no se pudo completar, se produjeron errores durante el "
"proceso de actualización."

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2035
msgid "Upgrade incomplete"
msgstr "Actualización incompleta"

#: ../DistUpgrade/DistUpgradeController.py:2036
msgid ""
"The upgrade has partially completed but there were errors during the upgrade "
"process."
msgstr ""
"La actualización se ha completado parcialmente pero se produjeron errores "
"durante el proceso de actualización."

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2058
#: ../DistUpgrade/DistUpgradeController.py:2109
#: ../DistUpgrade/DistUpgradeController.py:2115
#: ../DistUpgrade/DistUpgradeController.py:2126
msgid "Upgrade complete"
msgstr "Actualización completada"

#: ../DistUpgrade/DistUpgradeController.py:2059
#: ../DistUpgrade/DistUpgradeController.py:2110
#: ../DistUpgrade/DistUpgradeController.py:2116
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "La actualización se completó pero hubo errores durante el proceso."

#: ../DistUpgrade/DistUpgradeController.py:2078
msgid "System upgrade is complete."
msgstr "La actualización del sistema se ha completado."

#: ../DistUpgrade/DistUpgradeController.py:2127
msgid "The partial upgrade was completed."
msgstr "La actualización parcial se ha completado."

#: ../DistUpgrade/DistUpgradeController.py:2155
msgid "Lock screen disabled"
msgstr "Pantalla de bloqueo desactivada"

#: ../DistUpgrade/DistUpgradeController.py:2156
msgid ""
"Your lock screen has been disabled and will remain disabled until you reboot."
msgstr ""
"Su pantalla de bloqueo ha sido desactivada y seguirá así hasta que reinicie "
"el sistema."

#: ../DistUpgrade/DistUpgradeFetcher.py:107
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:170
msgid "Could not find the release notes"
msgstr "No se han podido encontrar las notas de publicación"

#: ../DistUpgrade/DistUpgradeFetcher.py:108
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:171
msgid "The server may be overloaded. "
msgstr "Puede que el servidor esté sobrecargado. "

#: ../DistUpgrade/DistUpgradeFetcher.py:120
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:175
msgid "Could not download the release notes"
msgstr "No se han podido descargar las notas de publicación"

#: ../DistUpgrade/DistUpgradeFetcher.py:121
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
msgid "Please check your internet connection."
msgstr "Por favor, compruebe su conexión a Internet."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:73
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "autenticar «%(file)s» contra «%(signature)s» "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:97
#, python-format
msgid "extracting '%s'"
msgstr "extrayendo «%s»"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:118
#: ../DistUpgrade/DistUpgradeFetcherCore.py:119
msgid "Could not run the upgrade tool"
msgstr "No se ha podido ejecutar la herramienta de actualización"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:120
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""
"Lo más probable es que esto sea un error en la herramienta de actualización. "
"Informe de esto con la orden «ubuntu-bug ubuntu-release-upgrader-core»."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:197
msgid "Upgrade tool signature"
msgstr "Firma de la herramienta de actualización"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:204
msgid "Upgrade tool"
msgstr "Herramienta de actualización"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:238
msgid "Failed to fetch"
msgstr "Error al descargar"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:239
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Ha fallado la descarga de la actualización. Puede haber un problema con la "
"red. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
msgid "Authentication failed"
msgstr "Error de autenticación"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:244
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Ha fallado la autenticación de la actualización. Es posible que exista un "
"problema con la red o el servidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
msgid "Failed to extract"
msgstr "Error al extraer"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:250
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Ha fallado la extracción de la actualización. Puede haber un problema con la "
"red o el servidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:255
msgid "Verification failed"
msgstr "Falló la verificación"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:256
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Ha fallado la verificación de la actualización. Puede haber un problema con "
"la red o con el servidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:270
#: ../DistUpgrade/DistUpgradeFetcherCore.py:276
msgid "Can not run the upgrade"
msgstr "No se ha podido ejecutar la actualización"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:271
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Esto normalmente ocurre en un sistema donde /tmp se ha montado como no "
"ejecutable. Vuelva a montarlo sin «noexec» y ejecute de nuevo la "
"actualización."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:277
#, python-format
msgid "The error message is '%s'."
msgstr "El mensaje de error es «%s»"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:151
msgid "&Upgrade"
msgstr "&Actualizar"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:154
#: ../DistUpgrade/DistUpgradeViewKDE.py:715
msgid "&Cancel"
msgstr "&Cancelar"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "Notas de la versión"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:191
msgid "Upgrade"
msgstr "Actualizar"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:207
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:222
msgid "Downloading additional package files..."
msgstr "Descargando archivos de paquetes adicionales..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:224
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Archivo %s de %s a %sB/s"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s"
msgstr "Archivo %s de %s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:235
#: ../DistUpgrade/DistUpgradeViewGtk3.py:125
#: ../DistUpgrade/DistUpgradeViewKDE.py:302
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Inserte «%s» en la unidad «%s»"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:237
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:242
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:243
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
msgid "Media Change"
msgstr "Cambio de soporte"

#: ../DistUpgrade/DistUpgradeQuirks.py:230
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr ""
"Su tarjeta gráfica puede no estar totalmente soportada en Ubuntu 14.04."

#: ../DistUpgrade/DistUpgradeQuirks.py:232
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D Do you still want to continue with the "
"upgrade?"
msgstr ""
"El entorno de escritorio en ejecución «Unity» no es totalmente compatible "
"con su hardware de gráficos. Tal vez terminé en un entorno muy lento después "
"de la actualización. Nuestro consejo es mantener la versión LTS, por ahora. "
"Para obtener más información, consulte https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D ¿Aún desea continuar con la actualización?"

#: ../DistUpgrade/DistUpgradeQuirks.py:256
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr ""
"Su hardware de gráficos no es plenamente compatible con Ubuntu 12.04 LTS."

#: ../DistUpgrade/DistUpgradeQuirks.py:258
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"La compatibilidad en Ubuntu 12.04 LTS para su hardware de gráficos Intel "
"está limitada y puede encontrar problemas tras la actualización. Para tener "
"más información vea https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForI8xx ¿Quiere continuar con la actualización?"

#: ../DistUpgrade/DistUpgradeQuirks.py:278
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"La actualización puede reducir los efectos de escritorio, así como el "
"rendimiento de los juegos y otros programas que usen gráficos de forma "
"intensiva."

#: ../DistUpgrade/DistUpgradeQuirks.py:282
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Este equipo está usando el controlador gráfico  «fglrx» de AMD. No se "
"encuentra disponible en Ubuntu 10.04 LTS una versión de este controlador que "
"funcione con su hardware.\n"
"\n"
"¿Quiere continuar?"

#: ../DistUpgrade/DistUpgradeQuirks.py:321
msgid "Sorry, no more upgrades for this system"
msgstr "Lo sentimos, no habrá más actualizaciones para este sistema"

#: ../DistUpgrade/DistUpgradeQuirks.py:322
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's 'i386' "
"architecture.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""
"No habrá más versiones de Ubuntu para la arquitectura «i386» que se usa en "
"este sistema.\n"
"\n"
"Las actualizaciones para Ubuntu %s seguirán hasta %s."

#: ../DistUpgrade/DistUpgradeQuirks.py:337
msgid "No ARMv6 CPU"
msgstr "No hay CPU ARMv6"

#: ../DistUpgrade/DistUpgradeQuirks.py:338
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Su sistema usa una CPU ARM que es más antigua que la arquitectura ARMv6. "
"Todos los paquetes en karmic han sido construidos con optimizaciones que "
"requieren ARMv6 como arquitectura mínima. No es posible actualizar sus "
"sistema a la nueva versión de Ubuntu con este hardware."

#: ../DistUpgrade/DistUpgradeQuirks.py:358
msgid "No init available"
msgstr "El demonio init no está disponible"

#: ../DistUpgrade/DistUpgradeQuirks.py:359
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Parece que su sistema es un entorno virtualizado sin un demonio init (un "
"Linux-VServer, p.ej.). Ubuntu 10.04 LTS no puede funcionar en este tipo de "
"entorno, por lo que primero se requiere una actualización de la "
"configuración de su máquina virtual.\n"
"\n"
"¿Está seguro de que quiere continuar?"

#: ../DistUpgrade/DistUpgradeQuirks.py:400
msgid "The Robot Operating System (ROS) is installed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:401
msgid ""
"It appears that ROS is currently installed. Each ROS release is very strict "
"about the versions of Ubuntu it supports, and Ubuntu upgrades can fail if "
"that guidance isn't followed. Before continuing, please either uninstall "
"ROS, or ensure the ROS release you have installed supports the version of "
"Ubuntu to which you're upgrading.\n"
"\n"
"For ROS 1 releases, refer to REP 3:\n"
"https://www.ros.org/reps/rep-0003.html\n"
"\n"
"For ROS 2 releases, refer to REP 2000:\n"
"https://www.ros.org/reps/rep-2000.html\n"
"\n"
"Are you sure you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:519
#, fuzzy
msgid "Connection to the Snap Store failed"
msgstr "La conexión con la tienda de snaps ha fallado"

#: ../DistUpgrade/DistUpgradeQuirks.py:520
msgid ""
"You have the package lxd installed but your system is unable to reach the "
"Snap Store. lxd is now provided via a snap and the release upgrade will fail "
"if snapd is not functional. Please make sure you're connected to the "
"Internet and update any firewall or proxy settings as needed so that you can "
"reach api.snapcraft.io. If you are an enterprise with a firewall setup you "
"may want to configure a Snap Store proxy."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:535
msgid "Connection to Snap Store failed"
msgstr "La conexión con la tienda de snaps ha fallado"

#: ../DistUpgrade/DistUpgradeQuirks.py:536
msgid ""
"Your system does not have a connection to the Snap Store. For the best "
"upgrade experience make sure that your system can connect to api.snapcraft."
"io.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"Su sistema no tiene un conexión con la tienda de snaps. Para obtener la "
"mejor experiencia con las actualizaciones asegúrese de que su sistema se "
"puede conectar a api.snapcraft.io.\n"
"¿Aún desea continuar con la actualización?"

#: ../DistUpgrade/DistUpgradeQuirks.py:545
msgid "Outdated snapd package"
msgstr "Paquete snapd desactualizado"

#: ../DistUpgrade/DistUpgradeQuirks.py:546
msgid ""
"Your system does not have the latest version of snapd. Please update the "
"version of snapd on your system to improve the upgrade experience.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"Su sistema no tiene la última versión de snapd. Actualice la versión de "
"snapd en su sistema para mejorar la experiencia de las actualizaciones.\n"
"¿Aún quiere continuar con la actualización?"

#. now perform direct API calls to the store, requesting size
#. information for each of the snaps needing installation
#: ../DistUpgrade/DistUpgradeQuirks.py:569
msgid "Calculating snap size requirements"
msgstr "Calculando los requisitos de tamaño de snap"

#: ../DistUpgrade/DistUpgradeQuirks.py:600
msgid "Processing snap replacements"
msgstr "Procesando reemplazos de snap"

#: ../DistUpgrade/DistUpgradeQuirks.py:606
#, python-format
msgid "refreshing snap %s"
msgstr "actualizando el snap %s"

#: ../DistUpgrade/DistUpgradeQuirks.py:610
#, fuzzy, python-format
msgid "removing snap %s"
msgstr "actualizando el snap %s"

#: ../DistUpgrade/DistUpgradeQuirks.py:613
#, python-format
msgid "installing snap %s"
msgstr "instalando el snap %s"

#: ../DistUpgrade/DistUpgradeQuirks.py:642
msgid "PAE not enabled"
msgstr "PAE no activado"

#: ../DistUpgrade/DistUpgradeQuirks.py:643
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""
"Su sistema usa una CPU que no tiene activado el PAE. Ubuntu solo soporta "
"sistemas no-PAE hasta Ubuntu 12.04. Para actualizar a una versión posterior "
"de Ubuntu debe activar el PAE (si es posible). Vea:\n"
"http://help.ubuntu.com/community/EnablingPAE"

#: ../DistUpgrade/DistUpgradeQuirks.py:976
msgid "Checking for installed snaps"
msgstr "Comprobando snaps instalados"

#: ../DistUpgrade/DistUpgradeMain.py:58
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr "Usar la ruta dada para buscar un CD-ROM con paquetes actualizables"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Usar interfaz de usuario. Actualmente disponibles: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid "*DEPRECATED* this option will be ignored"
msgstr "*OBSOLETO* esta opción se ignorará"

#: ../DistUpgrade/DistUpgradeMain.py:70
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Realizar solo una actualización parcial (no se reescribirá el «sources.list»)"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Disable GNU screen support"
msgstr "Desactive el soporte de pantalla GNU"

#: ../DistUpgrade/DistUpgradeMain.py:75
msgid "Set datadir"
msgstr "Establecer datadir"

#: ../DistUpgrade/DistUpgradeMain.py:78
msgid "Upgrade to the development release"
msgstr "Actualizar a la versión en desarrollo"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:146
#: ../DistUpgrade/DistUpgradeViewKDE.py:316
msgid "Fetching is complete"
msgstr "La descarga se ha completado"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:161
#: ../DistUpgrade/DistUpgradeViewKDE.py:332
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Descargando archivo %li de %li a %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:163
#: ../DistUpgrade/DistUpgradeViewGtk3.py:325
#: ../DistUpgrade/DistUpgradeViewKDE.py:333
#: ../DistUpgrade/DistUpgradeViewKDE.py:483
#, python-format
msgid "About %s remaining"
msgstr "Faltan alrededor de %s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:166
#: ../DistUpgrade/DistUpgradeViewKDE.py:335
#, python-format
msgid "Fetching file %li of %li"
msgstr "Descargando archivo %li de %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:198
#: ../DistUpgrade/DistUpgradeViewKDE.py:373
msgid "Applying changes"
msgstr "Aplicando los cambios"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:224
#: ../DistUpgrade/DistUpgradeViewKDE.py:386
msgid "dependency problems - leaving unconfigured"
msgstr "problemas de dependencias - se deja sin configurar"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:229
#: ../DistUpgrade/DistUpgradeViewKDE.py:388
#, python-format
msgid "Could not install '%s'"
msgstr "No se ha podido instalar «%s»"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:230
#: ../DistUpgrade/DistUpgradeViewKDE.py:389
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"La actualización continuará, pero es posible que el paquete «%s» no se "
"encuentre en un estado funcional. Considere la posibilidad de enviar un "
"informe de error acerca de esto."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:247
#: ../DistUpgrade/DistUpgradeViewKDE.py:411
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"¿Quiere sustituir el archivo de configuración modificado\n"
"«%s»?"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:248
#: ../DistUpgrade/DistUpgradeViewKDE.py:412
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Perderá todos los cambios que haya realizado en este archivo de "
"configuración si decide sustituirlo por una nueva versión."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:268
#: ../DistUpgrade/DistUpgradeViewKDE.py:435
msgid "The 'diff' command was not found"
msgstr "No se ha encontrado la orden «diff»"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:535
#: ../DistUpgrade/DistUpgradeViewText.py:140
msgid "A fatal error occurred"
msgstr "Ha ocurrido un error fatal"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:536
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Informe de este fallo (si todavía no lo ha hecho) e incluya los archivos /"
"var/log/dist-upgrade/main.log y /var/log/dist-upgrade/apt.log en el informe. "
"La actualización se ha cancelado.\n"
"Su archivo sources.list original se guardó en /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:566
msgid "Ctrl-c pressed"
msgstr "Se ha pulsado Ctrl-C"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:567
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Esto cancelará la operación y puede dejar al sistema en un estado "
"defectuoso. ¿Seguro que quiere hacer eso?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:706
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Para prevenir la pérdida de datos, cierre todas las aplicaciones y "
"documentos."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:720
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Ya no está soportado por Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:721
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Desactualizar (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:722
#, python-format
msgid "Remove (%s)"
msgstr "Eliminar (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:723
#, python-format
msgid "No longer needed (%s)"
msgstr "Ya no es necesario (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:724
#, python-format
msgid "Install (%s)"
msgstr "Instalar (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:725
#, python-format
msgid "Upgrade (%s)"
msgstr "Actualizar (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:773
msgid "Cancel"
msgstr "Cancelar"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:774
#: ../DistUpgrade/DistUpgradeViewKDE.py:939
msgid "Continue"
msgstr "Continuar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:447
msgid "Show Difference >>>"
msgstr "Mostrar diferencias >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:450
msgid "<<< Hide Difference"
msgstr "<<< Ocultar diferencias"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Error"
msgstr "Error"

#: ../DistUpgrade/DistUpgradeViewKDE.py:719
#: ../DistUpgrade/DistUpgradeViewKDE.py:960
msgid "&Close"
msgstr "&Cerrar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:753
msgid "Show Terminal >>>"
msgstr "Mostrar terminal >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:756
msgid "<<< Hide Terminal"
msgstr "<<< Ocultar terminal"

#: ../DistUpgrade/DistUpgradeViewKDE.py:829
msgid "Information"
msgstr "Información"

#: ../DistUpgrade/DistUpgradeViewKDE.py:877
msgid "&Start Upgrade"
msgstr "&Iniciar actualización"

#: ../DistUpgrade/DistUpgradeViewKDE.py:879
#: ../DistUpgrade/DistUpgradeViewKDE.py:923
#: ../DistUpgrade/DistUpgradeViewKDE.py:926
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Detalles"

#: ../DistUpgrade/DistUpgradeViewKDE.py:901
#, python-format
msgid "No longer supported %s"
msgstr "Ya no está soportado %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:903
#, python-format
msgid "Remove %s"
msgstr "Eliminar %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:905
#: ../DistUpgrade/DistUpgradeViewText.py:239
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Eliminar (fue autoinstalado) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:907
#, python-format
msgid "Install %s"
msgstr "Instalar %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:909
#, python-format
msgid "Upgrade %s"
msgstr "Actualizar %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
#: ../DistUpgrade/DistUpgradeViewText.py:292
msgid "Restart required"
msgstr "Se requiere reiniciar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Reinicie el sistema para completar la actualización</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:959
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "_Reiniciar ahora"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:977
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>¿Cancelar la actualización en curso?</big></b>\n"
"\n"
"El sistema podría quedar en un estado no utilizable si cancela la "
"actualización. Le recomendamos encarecidamente que continúe la actualización."

#: ../DistUpgrade/DistUpgradeViewKDE.py:981
msgid "Cancel Upgrade?"
msgstr "¿Cancelar la actualización?"

#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li día"
msgstr[1] "%li días"

#: ../DistUpgrade/DistUpgradeView.py:84
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li hora"
msgstr[1] "%li horas"

#: ../DistUpgrade/DistUpgradeView.py:86
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minuto"
msgstr[1] "%li minutos"

#: ../DistUpgrade/DistUpgradeView.py:87
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li segundo"
msgstr[1] "%li segundos"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:103
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:121
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:172
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Esta descarga tardará %s aproximadamente con una conexión DSL de 1Mbit y %s "
"aproximadamente con un módem de 56k."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:176
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Esta descarga tardará aproximadamente %s con su conexión actual. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:270 ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Preparing to upgrade"
msgstr "Preparando la actualización"

#: ../DistUpgrade/DistUpgradeView.py:271
msgid "Getting new software channels"
msgstr "Obteniendo nuevos canales de software"

#: ../DistUpgrade/DistUpgradeView.py:272 ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Getting new packages"
msgstr "Obteniendo paquetes nuevos"

#: ../DistUpgrade/DistUpgradeView.py:273 ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Installing the upgrades"
msgstr "Instalando las actualizaciones"

#: ../DistUpgrade/DistUpgradeView.py:274 ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Cleaning up"
msgstr "Limpiando"

#: ../DistUpgrade/DistUpgradeView.py:363
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d paquete instalado ya no está soportado por Canonical. Puede "
"seguir obteniendo soporte de la comunidad."
msgstr[1] ""
"%(amount)d paquetes instalados ya no están soportados por Canonical. Puede "
"seguir obteniendo soporte de la comunidad."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:372
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "Se va a desinstalar %d paquete."
msgstr[1] "Se van a desinstalar %d paquetes."

#: ../DistUpgrade/DistUpgradeView.py:377
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "Se va a instalar %d paquete nuevo."
msgstr[1] "Se van a instalar %d paquetes nuevos."

#: ../DistUpgrade/DistUpgradeView.py:383
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "Se va a actualizar %d paquete."
msgstr[1] "Se van a actualizar %d paquetes."

#: ../DistUpgrade/DistUpgradeView.py:391
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Debe descargar un total de %s. "

#: ../DistUpgrade/DistUpgradeView.py:396
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"Esta actualización puede tardar varias horas. Una vez finalice la descarga, "
"el proceso no se podrá cancelar."

#: ../DistUpgrade/DistUpgradeView.py:400
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"Recoger e instalar la actualización puede llevar varias horas. Una vez que "
"la descarga haya terminado, el proceso no se puede cancelar."

#: ../DistUpgrade/DistUpgradeView.py:405
msgid "Removing the packages can take several hours. "
msgstr "La desinstalación de los paquetes puede tardar varias horas. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:410
msgid "The software on this computer is up to date."
msgstr "El software de este equipo está actualizado."

#: ../DistUpgrade/DistUpgradeView.py:411
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"No hay actualizaciones disponibles para su sistema. Se ha cancelado la "
"actualización."

#: ../DistUpgrade/DistUpgradeView.py:427
msgid "Reboot required"
msgstr "Se requiere reiniciar el equipo"

#: ../DistUpgrade/DistUpgradeView.py:428
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"La actualización ha finalizado y se necesita reiniciar el equipo. ¿Quiere "
"hacerlo ahora?"

#: ../DistUpgrade/DistUpgradeViewText.py:82
#, python-format
msgid "Progress: [%3i%%]"
msgstr "Progreso: [%3i %%]"

#: ../DistUpgrade/DistUpgradeViewText.py:141
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Informe de este fallo e incluya los archivos /var/log/dist-upgrade/main.log "
"y /var/log/dist-upgrade/apt.log en el informe. La actualización ha sido "
"cancelada.\n"
"Su archivo sources.list original se guardó en /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "Aborting"
msgstr "Cancelando"

#: ../DistUpgrade/DistUpgradeViewText.py:171
msgid "Demoted:\n"
msgstr "Para quitar:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
msgid "To continue please press [ENTER]"
msgstr "Para continuar, pulse Intro"

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
#: ../DistUpgrade/DistUpgradeViewText.py:261
msgid "Continue [yN] "
msgstr "Continuar [sN] "

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
msgid "Details [d]"
msgstr "Detalles [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:219
#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "y"
msgstr "s"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:222
#: ../DistUpgrade/DistUpgradeViewText.py:271
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:230
#, python-format
msgid "No longer supported: %s\n"
msgstr "Ya no está soportado: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:235
#, python-format
msgid "Remove: %s\n"
msgstr "Desinstalar: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:245
#, python-format
msgid "Install: %s\n"
msgstr "Instalar: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:250
#, python-format
msgid "Upgrade: %s\n"
msgstr "Actualizar: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:268
msgid "Continue [Yn] "
msgstr "Continuar [Sn] "

#: ../DistUpgrade/DistUpgradeViewText.py:293
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Para finalizar la actualización se necesita reiniciar.\n"
"Si selecciona «s» el sistema reiniciará."

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Descargando archivo %(current)li de %(total)li a %(speed)s/s"

#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Descargando archivo %(current)li de %(total)li"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "Mostrar el progreso de cada archivo individual"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Cancelar la actualización"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "_Continuar la actualización"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>¿Cancelar la actualización en curso?</big></b>\n"
"\n"
"El sistema podría quedar en un estado no utilizable si cancela la "
"actualización. Le recomendamos encarecidamente que continúe la actualización."

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Iniciar la actualización"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Sustituir"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Diferencia entre los archivos"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_Informar de un error"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Continuar"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>¿Comenzar la actualización?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Reinicie el sistema para completar la actualización</big></b>\n"
"\n"
"Guarde su trabajo antes de continuar."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Actualización completa de la distribución"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
#, no-c-format
msgid "<b><big>Upgrading Ubuntu to version %s</big></b>"
msgstr "<b><big>Actualizando Ubuntu a la versión %s</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Setting new software channels"
msgstr "Configurando nuevos canales de software"

#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Restarting the computer"
msgstr "Reiniciando el equipo"

#: ../data/gtkbuilder/DistUpgrade.ui.h:28
msgid "Terminal"
msgstr "Terminal"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "_Actualizar"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr ""
"<b>Está disponible una nueva versión de Ubuntu. ¿Le gustaría actualizar?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "No actualizar"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "Preguntar más tarde"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "Sí, actualizar ahora"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Ha decidido no actualizarse a la nueva versión de Ubuntu"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"Puede actualizar posteriormente yendo a la Actualización de software y "
"pulsando en «Actualizar»."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr "Realizar una actualización de la versión"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr "Para actualizar Ubuntu, necesita autenticarse."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr "Realizar una actualización parcial"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr "Para realizar una actualización parcial necesita autenticarse."

#: ../do-partial-upgrade:75 ../do-release-upgrade:80
msgid "Show version and exit"
msgstr "Mostrar la versión y salir"

#: ../do-partial-upgrade:78 ../do-release-upgrade:87
msgid "Directory that contains the data files"
msgstr "Directorio que contiene los archivos de datos"

#: ../do-partial-upgrade:81 ../do-release-upgrade:100
msgid "Run the specified frontend"
msgstr "Ejecutar la interfaz especificada"

#: ../do-partial-upgrade:112
msgid "Running partial upgrade"
msgstr "Ejecutando una actualización parcial"

#: ../do-release-upgrade:34 ../do-release-upgrade:46
msgid "Downloading the release upgrade tool"
msgstr "Descargando la herramienta de actualización de la versión"

#: ../do-release-upgrade:83
msgid ""
"If using the latest supported release, upgrade to the development release"
msgstr ""
"Si utiliza la última versión admitida, actualice a la versión de desarrollo"

#: ../do-release-upgrade:90 ../check_new_release_gtk.py:181
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Intente actualizar a la última versión usando el actualizador de $distro-"
"proposed"

#: ../do-release-upgrade:94
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Ejecutar en un modo especial de actualización.\n"
"Actualmente se permiten los modos «desktop» (para actualizaciones normales "
"de un sistema de escritorio) y «server» (para servidores)."

#: ../do-release-upgrade:103
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Comprueba únicamente si está disponible una nueva versión de la distribución "
"e informa del resultado mediante un código de salida"

#: ../do-release-upgrade:108
msgid ""
"Try the upgrade with third party mirrors and repositories enabled instead of "
"commenting them out."
msgstr ""
"Intente la actualización con espejos y repositorios de terceros activados en "
"vez de quitarlos en forma de comentario."

#: ../do-release-upgrade:122
msgid "The options --devel-release and --proposed are"
msgstr "Las opciones --devel-release y --proposed son"

#: ../do-release-upgrade:123
msgid "mutually exclusive. Please use only one of them."
msgstr "mutuamente exclusivas. Use solo una de ellas."

#: ../do-release-upgrade:127
msgid "Checking for a new Ubuntu release"
msgstr "Comprobar si hay una nueva versión de Ubuntu"

#: ../do-release-upgrade:139
msgid "Your Ubuntu release is not supported anymore."
msgstr "Su versión de Ubuntu ya no tiene soporte."

#: ../do-release-upgrade:140
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Para saber más sobre esta actualización, visite:\n"
"%(url)s\n"

#: ../do-release-upgrade:147
msgid "In /etc/update-manager/release-upgrades Prompt "
msgstr "En indicador /etc/update-manager/release-upgrades "

#: ../do-release-upgrade:148
msgid "is set to never so upgrading is not possible."
msgstr "está establecido en nunca por lo que actualizar no es posible."

#: ../do-release-upgrade:150
msgid "There is no development version of an LTS available."
msgstr "No hay una versión de desarrollo de una LTS disponible."

#: ../do-release-upgrade:151
msgid "To upgrade to the latest non-LTS development release "
msgstr "Para actualizar al último lanzamiento de desarrollo sin LTS "

#: ../do-release-upgrade:152
msgid "set Prompt=normal in /etc/update-manager/release-upgrades."
msgstr "defina «Prompt=normal» en /etc/update-manager/release-upgrades."

#: ../do-release-upgrade:154
msgid "Upgrades to the development release are only "
msgstr "Las actualizaciones de la versión de desarrollo están "

#: ../do-release-upgrade:155
msgid "available from the latest supported release."
msgstr "disponibles desde la última versión soportada."

#: ../do-release-upgrade:157
msgid "No new release found."
msgstr "No hay ninguna versión nueva."

#: ../do-release-upgrade:162
msgid "Release upgrade not possible right now"
msgstr "En este momento no es posible la actualización de versión"

#: ../do-release-upgrade:163
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"La actualización de versión no se puede realizar en este momento, inténtelo "
"de nuevo después. El servidor informó: «%s»"

#: ../do-release-upgrade:169
#, c-format
msgid "New release '%s' available."
msgstr "Está disponible la nueva versión «%s»."

#: ../do-release-upgrade:170
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Ejecute «do-release-upgrade» para actualizarse a ella."

#: ../do-release-upgrade:187
msgid "Please install all available updates for your release before upgrading."
msgstr ""
"Instale todas las actualizaciones disponibles de su versión antes de "
"actualizar la distribución."

#: ../do-release-upgrade:203
msgid ""
"You have not rebooted after updating a package which requires a reboot. "
"Please reboot before upgrading."
msgstr ""
"No ha reiniciado después de actualizar un paquete que requiere un reinicio. "
"Reinicie antes de actualizar."

#: ../check_new_release_gtk.py:85
msgid "Software Updater"
msgstr "Actualización de software"

#: ../check_new_release_gtk.py:113
#, python-format
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Está disponible la actualización a Ubuntu %(version)s"

#. print("don't upgrade")
#: ../check_new_release_gtk.py:144
#, python-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Ha decidido no actualizarse a Ubuntu %s"

#: ../check_new_release_gtk.py:177
msgid "Check if upgrading to the latest devel release is possible"
msgstr "Comprobar si es posible actualizar a la última versión de desarrollo"

#: ../check_new_release_gtk.py:186
msgid "Add debug output"
msgstr "Añadir resultado de la depuración"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade:\n"
#~ "%s\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "Ha ocurrido un error irresoluble mientras se calculaba la actualización:\n"
#~ "%s\n"
#~ "\n"
#~ " Esto puede deberse a:\n"
#~ " * Que se está actualizando a una versión de Ubuntu aún no publicada\n"
#~ " * Que se está usando la actual versión aún no publicada de Ubuntu\n"
#~ " * Paquetes de software no oficiales, no suministrados por Ubuntu\n"
#~ "\n"

#~ msgid ""
#~ "This upgrade is running in sandbox (test) mode. All changes are written "
#~ "to '%s' and will be lost on the next reboot.\n"
#~ "\n"
#~ "*No* changes written to a system directory from now until the next reboot "
#~ "are permanent."
#~ msgstr ""
#~ "Esta actualización se esta realizando en el modo (prueba) «sandbox». "
#~ "Todos los cambios se escriben en «%s» y se perderán en el siguiente "
#~ "arranque.\n"
#~ "\n"
#~ "*Ninguno* de los cambios escritos en el directorio de sistema desde ahora "
#~ "hasta el siguiente arranque son permanentes."

#~ msgid ""
#~ "The upgrade can not continue with that package installed.\n"
#~ "Please remove it with synaptic or 'apt-get remove debsig-verify' first "
#~ "and run the upgrade again."
#~ msgstr ""
#~ "La actualización no puede continuar con ese paquete instalado.\n"
#~ "Desinstálelo con synaptic o «apt-get remove debsig-verify» primero y "
#~ "luego intente actualizar nuevamente."

#~ msgid ""
#~ "While scanning your repository information no mirror entry for the "
#~ "upgrade was found. This can happen if you run a internal mirror or if the "
#~ "mirror information is out of date.\n"
#~ "\n"
#~ "Do you want to rewrite your 'sources.list' file anyway? If you choose "
#~ "'Yes' here it will update all '%s' to '%s' entries.\n"
#~ "If you select 'No' the upgrade will cancel."
#~ msgstr ""
#~ "Mientras se exploraba la información de su repositorio, no se encontró "
#~ "ninguna entrada para la réplica de la actualización. Esto puede ocurrir "
#~ "si corre una réplica interna o si la información de la réplica es "
#~ "antigua.\n"
#~ "\n"
#~ "¿Quiere reescribir su archivo «sources.list» de todos modos? Si elige "
#~ "«Sí» se actualizarán todas las entradas «%s» a «%s».\n"
#~ "Si selecciona «No» la actualización se cancelará."

#~ msgid ""
#~ "The upgrade has aborted. The upgrade needs a total of %s free space on "
#~ "disk '%s'. Please free at least an additional %s of disk space on '%s'. "
#~ "Empty your trash and remove temporary packages of former installations "
#~ "using 'sudo apt-get clean'."
#~ msgstr ""
#~ "La actualización se canceló. La actualización necesita un total de %s de "
#~ "espacio libre en el disco «%s». Libere al menos %s de espacio en el disco "
#~ "«%s». Pruebe vaciando su papelera y borrando paquetes temporales de "
#~ "antiguas instalaciones usando la orden «sudo apt-get clean»."

#~ msgid "evms in use"
#~ msgstr "Se está usando «evms»"

#~ msgid ""
#~ "Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
#~ "software is no longer supported, please switch it off and run the upgrade "
#~ "again when this is done."
#~ msgstr ""
#~ "Su sistema está usando el gestor de volúmenes «evms» en /proc/mounts. El "
#~ "software «evms» ya no está soportado; por favor, desactívelo y vuelva a "
#~ "ejecutar de nuevo la actualización."

#~ msgid ""
#~ "This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
#~ "version of this driver is available that works with your video card in "
#~ "Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "Este equipo está usando el controlador gráfico  «nvidia» de NVIDIA. No se "
#~ "encuentra disponible en Ubuntu 10.04 LTS una versión de este controlador "
#~ "que funcione con su hardware.\n"
#~ "\n"
#~ "¿Quiere continuar?"

#~ msgid "No i686 CPU"
#~ msgstr "Sin CPU i686"

#~ msgid ""
#~ "Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
#~ "extension. All packages were built with optimizations requiring i686 as "
#~ "the minimal architecture. It is not possible to upgrade your system to a "
#~ "new Ubuntu release with this hardware."
#~ msgstr ""
#~ "Su sistema usa una CPU i586 CPU o  una CPU que no tiene la extensión "
#~ "«cmov». Todos los paquetes se han construido con optimizaciones que "
#~ "requieren i686 como arquitectura mínima. No es posible actualizar su "
#~ "sistema a la nueva versión de Ubuntu con este hardware."

#~ msgid "Sandbox upgrade using aufs"
#~ msgstr "Actualización de prueba usando aufs"

#~ msgid "No new release found"
#~ msgstr "No se ha encontrado ninguna edición nueva"

#~ msgid "Authentication is required to perform a partial upgrade"
#~ msgstr "Para realizar una actualización parcial, necesita autenticarse"

#~ msgid "Authentication is required to perform a release upgrade"
#~ msgstr "Para actualizar la distribución, necesita autenticarse"

#~ msgid ""
#~ "If none of this applies, then please report this bug using the command "
#~ "'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
#~ msgstr ""
#~ "Si nada de esto aplica, informe de este error usando la orden «ubuntu-bug "
#~ "ubuntu-release-upgrader-core» en una terminal."

#~ msgid "Your graphics hardware may not be fully supported in Ubuntu 13.04."
#~ msgstr ""
#~ "Su hardware de gráficos podría no ser compatible completamente con Ubuntu "
#~ "13.04."

#~ msgid "<b><big>Upgrading Ubuntu to version 13.04</big></b>"
#~ msgstr "<b><big>Actualizando Ubuntu a la versión 13.04</big></b>"

#~ msgid "Package 'debsig-verify' is installed"
#~ msgstr "El paquete «debsig-verify» está instalado"

#~ msgid "Test upgrade with a sandbox aufs overlay"
#~ msgstr "Comprobar la actualización en una capa aufs de prueba"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade.\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "Ha ocurrido un problema mientras se calculaba la actualización.\n"
#~ "\n"
#~ " Eso se ha podido causar por:\n"
#~ " * Actualizar a una versión de Ubuntu aún no publicada\n"
#~ " * Estar ejecutando la versión actual, aún no publicada, de Ubuntu\n"
#~ " * Paquetes no oficiales de software no proporcionados por Ubuntu\n"
#~ "\n"

#~ msgid ""
#~ "Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop or edubuntu-desktop package and it was not possible to detect "
#~ "which version of Ubuntu you are running.\n"
#~ " Please install one of the packages above first using synaptic or apt-get "
#~ "before proceeding."
#~ msgstr ""
#~ "Su sistema no contiene el paquete ubuntu-desktop,  kubuntu-desktop, "
#~ "xubuntu-desktop o edubuntu-desktop, y no ha sido posible detectar qué "
#~ "versión de Ubuntu está ejecutando.\n"
#~ " Instale uno de los paquetes anteriores usando Synaptic o apt-get antes "
#~ "de proceder."

#~ msgid "Sandbox mode"
#~ msgstr "Modo de prueba"

#~ msgid "It was not possible to create the sandbox environment."
#~ msgstr "No ha sido posible crear un entorno de prueba."

#~ msgid "Sandbox setup failed"
#~ msgstr "Falló la configuración de prueba"

#~ msgid ""
#~ "Your python install is corrupted. Please fix the '/usr/bin/python' "
#~ "symlink."
#~ msgstr ""
#~ "Su instalación de python está dañada. Repare el enlace simbólico «/usr/"
#~ "bin/python»."

#~ msgid "<b><big>Upgrading Ubuntu to version 15.04</big></b>"
#~ msgstr "<b><big>Actualizando Ubuntu a la versión 15.04</big></b>"

#~ msgid "<b><big>Upgrading Ubuntu to version 15.10</big></b>"
#~ msgstr "<b><big>Actualizando Ubuntu a la versión 15.10</big></b>"

#~ msgid "apt architecture is i386"
#~ msgstr "la arquitectura de apt es i386"

#~ msgid ""
#~ "Upgrades from 18.04 on the i386 architecture are not supported at this "
#~ "time."
#~ msgstr ""
#~ "Las actualizaciones de 18.04 en la arquitectura i386 no se admiten en "
#~ "este momento."
